### PR TITLE
Task-38524_38523: Display missing external label

### PR DESCRIPTION
**Pull request by ahmadmido**
This PR will add the external tag in exoUserAvatar and exoSuggesterUser.

### CODE REVIEW

File: webapp/portlet/src/main/webapp/common/components/ExoUserAvatar.vue [hunk 1]
```diff
@@ -87,8 +87,22 @@ export default {
...
   },
+  computed: {
+    externalFullName() {
+      return this.fullname.concat(' (').concat(this.$t('external.tag')).concat(')');
+    },
+  },
+  created() {
+    this.$userService.getUser(this.username)
+      .then(user => {
+        if(user.external === 'true'){
```
Comment (azayati) [inline webapp/portlet/src/main/webapp/common/components/ExoUserAvatar.vue]:
```suggestion
         this.isExternal = user.external !== undefined && user.external === 'true'
```

File: webapp/portlet/src/main/resources/locale/portlet/Portlets_en.properties [hunk 2]
```diff
@@ -465,3 +465,8 @@ search.connector.label.clear=Clear
 #                                    Textarea                                       #
 #####################################################################################
 textarea.limitMessage=Only {0} characters allowed
+
+#####################################################################################
+#                                    External                                       #
+#####################################################################################
+external.tag=External
```
Comment (azayati) [inline webapp/portlet/src/main/resources/locale/portlet/Portlets_en.properties]:
```suggestion
userAvatar.external.label=External
```

**Review by azayati — CHANGES_REQUESTED** [review]
See review

File: webapp/portlet/src/main/webapp/common/components/ExoIdentitySuggester.vue [hunk 3]
```diff
@@ -54,7 +54,7 @@
             <v-img :src="item.profile.avatarUrl"></v-img>
           </v-avatar>
           <span class="text-truncate">
-            {{ item.profile.fullName }}
+            {{ item.profile.external !== null && item.profile.external ? item.profile.fullName.concat(' (').concat($t('userAvatar.external.label')).concat(')') : item.profile.fullName }}
```
Comment (azayati) [inline webapp/portlet/src/main/webapp/common/components/ExoIdentitySuggester.vue]: Is it possible to compute it ?

**Review by azayati — CHANGES_REQUESTED** [review]
see review

File: webapp/portlet/src/main/webapp/common/components/ExoIdentitySuggester.vue (hunk 3 above)
Comment (ahmadmido) [reply webapp/portlet/src/main/webapp/common/components/ExoIdentitySuggester.vue]: It's not possible to use compute, in this case, it's related to item the external and not to items witch is declared.

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**